iSkyLIMS_drylab/views.py

```diff
@@ -417,19 +475,20 @@ def search_service(request):
     services_search_list = {}
 
     center_list_abbr = []
-    center_availables = Center.objects.all().order_by("centerAbbr")
+    center_availables = iSkyLIMS_drylab.models.Center.objects.all().order_by(
```

Comment:
Center class does not belongs to iSkyLIMS_drylab but to django_utils 
Change to 
django_utils.models.Center